browser/src/control/Control.PartsPreview.js

```diff
@@ -226,8 +226,18 @@ L.Control.PartsPreview = L.Control.extend({
 			}
 			if (app.file.fileBasedView)
 				this._map._docLayer._checkSelectedPart();
+			img.focus();
```

Comment:
What about this piece, do we need this img.focus() change?

Reply:
yes, otherwise we wouldn't know if we need to dispatch copySlide or normal Copy. Currently we copy slide if slide has focus. Its only focused on click.

Any element can only have focus if it has tabindex.

Reply:
got it, thanks